### ISSUE TITLE
Cascade deleting station

### ISSUE THREAD

**Issue by stanislawpolanski** [stanislawpolanski/MrnWebApi#73]
Add cascade deleting in the database (new task etc).

**Comment by stanislawpolanski** [comment]
https://github.com/stanislawpolanski/MalopolskaRailNetworkWwwSqlDatabase/issues/50

**Comment by stanislawpolanski** [comment]
Deleting stations will be done in the app.

**Comment by stanislawpolanski** [comment]
Blocked by #81

**Comment by stanislawpolanski** [comment]
Move to database layer eventually.

**Comment by stanislawpolanski** [comment]
Fixed in db layer

**Comment by stanislawpolanski** [comment]
https://github.com/stanislawpolanski/MalopolskaRailNetworkWwwSqlDatabase/issues/50
